src/main/flight/pid.c

```diff
@@ -1130,3 +1176,27 @@ bool pidAntiGravityEnabled(void)
 {
     return antiGravityEnabled;
 }
+
+#ifdef USE_DYN_LPF
+void dynLpfDTermUpdate(float throttle)
+{
+    if (dynLpfFilter == DYN_LPF_PT1 || dynLpfFilter == DYN_LPF_BIQUAD) {
+        uint16_t cutoffFreq = dynLpfMin;
+        if (throttle > dynLpfIdle) {
+            const float dynThrottle = (throttle - (throttle * throttle * throttle) * 0.333f) * 1.5f;
+            cutoffFreq += (dynThrottle - dynLpfIdlePoint) * dynLpfInvIdlePoint * dynLpfDiff;
+         }
+
+         if (dynLpfFilter == DYN_LPF_PT1) {
+            const float gyroDt = gyro.targetLooptime * 1e-6f;
```

Comment:
this calculation is wrong because mixer is not running in gyro.targetLooptime but at targetPidLooptime whish is
gyro.targetLooptime * pidConfig()->pid_process_denom